types/cmd-shim/index.d.ts

```diff
@@ -1,30 +1,26 @@
-// Type definitions for cmd-shim 2.0
-// Project: https://github.com/ForbesLindesay/cmd-shim
+// Type definitions for cmd-shim 5.0
+// Project: https://github.com/npm/cmd-shim
 // Definitions by: Andrew Bradley <https://github.com/cspotcode>
+//                 Hiroki Osame <https://github.com/privatenumber>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /**
- *
  * Create a cmd shim at `to` for the command line program at `from`. e.g.
  *
- *     var cmdShim = require('cmd-shim');
- *     cmdShim(__dirname + '/cli.js', '/usr/bin/command-name', function (err) {
- *         if (err) throw err;
- *     });
+ *     import cmdShim from 'cmd-shim';
```

Comment:
Only question I have is about these example imports:

It says `import cmdShim from`, but the tests do `import cmdShim = require(`

Is there a difference?

Reply:
I marked my review as "request changes" to make the bot happy.  Changes may not be necessary; just my question about the `import`s

Reply:
It's still a CJS package so I guess technically it's `import cmdShim = require(` unless `esModuleInterop` is enabled.

Nowadays though, I think it's rare to write in `import = require()` syntax, so I think it's more counter intuitive.

Reply:
What do you think about removing the import statement entirely? It's user-setup specific so might make sense not to include it.

For reference, another type I contributed to doesn't include it: https://unpkg.com/@types/cheerio@0.22.31/index.d.ts

Reply:
I went ahead and removed the imports.

Hope this works with you.

Reply:
Sounds good.  I trust your judgement; whatever you think is best, I'll approve the PR.